payments-core/src/main/java/com/stripe/android/view/PaymentAuthWebViewActivity.kt

```diff
@@ -79,10 +81,12 @@ class PaymentAuthWebViewActivity : AppCompatActivity() {
 
         logger.debug("PaymentAuthWebViewActivity#onCreate() - PaymentAuthWebView init and loadUrl")
 
-        val isPagedLoaded = MutableLiveData(false)
-        isPagedLoaded.observe(this) { shouldHide ->
-            if (shouldHide) {
-                viewBinding.progressBar.isGone = true
+        val isPagedLoaded = MutableStateFlow(false)
```

Comment:
I know this is a direct conversion, but should we consider moving this to the ViewModel?

Reply:
I think it's in the right spot. We wouldn't want to store the loading state of a view in a view model.

This loading is related to the WebViewClient, which lasts the same duration as the activity here.